safenode/src/client/api.rs

```diff
@@ -113,7 +113,13 @@ impl Client {
                 self.events_channel
                     .broadcast(ClientEvent::ConnectedToNetwork);
 
-                let key = NetworkAddress::from_peer(peer_id);
+                let target = {
```

Comment:
To me this seems like we are trying to populate our RT in some fakish way. 
If we connect to any node then get closest to us then we can connect to the results of that. 
Our RT will be populated when we do spend parent checking or any other network search. 
IMO all we need to do initially is connect to the peers close to our address.